projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -711,8 +711,11 @@ public static function enqueue_block_editor_assets() {
 		if ( Jetpack::is_module_active( 'publicize' ) && function_exists( 'publicize_init' ) ) {
 			$publicize               = publicize_init();
 			$initial_state['social'] = array(
-				'sharesData'  => $publicize->get_publicize_shares_info( $blog_id ),
-				'hasPaidPlan' => $publicize->has_paid_plan(),
+				'sharesData'              => $publicize->get_publicize_shares_info( $blog_id ),
+				'hasPaidPlan'             => $publicize->has_paid_plan(),
+				'publicizeConnectionsUrl' => esc_url_raw(
+					'https://jetpack.com/redirect/?source=jetpack-social-connections-block-editor&site='
```

Comment:
I wonder if we should leave this as having the default of https://wordpress.com/marketing/connections because this will be used on WPCOM too, and it seems odd to send them via a Jetpack redirect.

Moving it to the usePublicizeConfig hook seems useful though, so it can be used in various places, rather than just the share panel.

It's also taking me to wordpress.com rather than cloud.jetpack.com when I'm using Jetpack Social